access-grant/src/main/java/com/inrupt/client/accessgrant/AccessGrantClient.java

```diff
@@ -544,7 +544,9 @@ public CompletionStage<Void> revoke(final AccessCredential credential) {
      *
      * @param credential the access credential
      * @return the next stage of completion
+     * @deprecated as of Beta4
      */
+    @Deprecated
```

Comment:
Can you move this into its own PR with a dedicated ticket?

Reply:
done